docs/src/main/sphinx/sql/set-time-zone.rst

```diff
@@ -0,0 +1,61 @@
+=============
+SET TIME ZONE
+=============
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET TIME ZONE LOCAL
+    SET TIME ZONE 'string'
+    SET TIME ZONE INTERVAL interval
```

Comment:
This is no longer accurate. It supports any expression that returns a string value or an interval value.